package.json

```diff
@@ -33,10 +32,13 @@
     "grunt-jscs": "^2.1.0",
     "grunt-newer": "*",
     "grunt-webpack": "^1.0.11",
+    "jquery": "^2.1.4",
     "load-grunt-config": "*",
+    "lodash": "^4.6.1",
+    "node-sass": "^3.4.2",
```

Comment:
If I've missed something in the Harp docs please point me in the right direction. I would much rather rely on Harp then bring in another dependency.